joomla/joomla-cms
# [4.0] Group toolbar buttons for links

### Summary of Changes
This PR continues the grouping of action buttons in the backend (see #22832, #22393, #23337 and #23336, #23374, #23368, #23380). This time the reditect links.

### Testing Instructions

Go to the redirect links in the backend and check the toolbar. There should be a "Change status" dropdown now which activates when you select a redirect link.

NOTE: The dropdown contains the status archived, as in 3.x. 
This status is not selectable in the status filter. But this another issue and not in scope of this PR.
